src/jquery.contextMenu.js

```diff
@@ -1227,7 +1234,7 @@
                                 createNameNode(item).appendTo($t);
 
                                 item.appendTo = item.$node;
-                                op.create(item, root);
+                                //op.create(item, root); decide later, might be a promise.
```

Comment:
Why isn't the logic for creating the submenu called here but evaluated later, outside the case statement?